easybuild/easyconfigs/s/structure/structure-2.3.4.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild recipy by Christian Meesters (2017)
+# HPC-group University of Mainz (https://hpc.uni-mainz.de)
+
+easyblock = 'MakeCp'
+name = 'structure'
+version = '2.3.4'
+
+homepage = 'https://web.stanford.edu/group/pritchardlab/structure.html'
+description = """The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include infering the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. It can be applied to most of the commonly-used genetic markers, including SNPS, microsatellites, RFLPs and AFLPs."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://web.stanford.edu/group/pritchardlab/structure_software/release_versions/v%(version)s/']
+sources     = ['structure_kernel_source.tar.gz']
+files_to_copy = ['structure']
+
+sanity_check_paths = {'files' : ['structure'],
+                      'dirs'  : []}
+#buildininstalldir = True
+
+#installopts = 'PREFIX=%(installdir)s'
+moduleclass = 'bio'
+
+
+
+allow_prepend_abs_path = True
+modextrapaths = {'PATH': '%(installdir)s' }
```

Comment:
you don't need `allow_prepend_abs_path = True` here, just use:

```python
modextrapaths = {'PATH': ''}
```

(the installation prefix is prepended automatically for all entries)

Also, please move this up, above the `moduleclass = ...` line